web/src/utils/index.js

```diff
@@ -48,7 +48,7 @@ function copyToClipboard(text) {
 }
 
 function getDandisetContact(dandiset) {
-  if (dandiset.meta.dandiset.contributors) {
+  if (dandiset.meta?.dandiset.contributors) {
```

Comment:
Shouldn't this `meta` be removed? Is it related to the fact that this feature just doesn't work correctly right now?

Reply:
Yes. If the optional chaining operator isn't there then it crashes and the page won't load. This is fixed by #783, which is set to merge into this PR's branch but I wanted to hold off on that so its code doesn't show up in the diff here.